extensions/industry/healthcare/fhir/fieldgroups/immunization.schema.json

```diff
@@ -0,0 +1,297 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.fhir.org/fieldgroup/immunization",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Definition of a immunization",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:intendedToExtend":[ ],
```

Comment:
Since we don't know which class it will be used so Bisin Moothandasseril recommended to use this option.
Is there another way to leave it open ended?